recipes/pygrametl/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "2.5.0" %}
+{% set name = "pygrametl" %}
+
+package:
+  name: pygrametl
+  version: {{ version }}
+
+source:
+  fn: pygrametl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7db0b98f6c817fc39e3d049c74b6aafbe0807e9322565fca1acdf9cf83607ad1
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - pygrametl
+
+about:
+  home: http://pygrametl.org/
+  license: BSD 2-Clause
+  license_file: LICENSE.txt
+  license_family: BSD
+  summary: 'ETL programming in Python'
+
+extra:
+  recipe-maintainers:
+    - ericdill
+    - mariusvniekerk
```

Comment:
Just to confirm, is it ok for you to be a maintainer here, @mariusvniekerk? If you just want to give blanket permission for @ericdill to add you to recipes (or within some guidelines), please feel free to do that in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ).